network/src/peers/peers.rs

```diff
@@ -425,39 +427,18 @@ impl<S: Storage + Send + Sync + 'static> Node<S> {
         }
     }
 
-    /// TODO (howardwu): Implement manual serializers and deserializers to prevent forward breakage
-    ///  when the PeerBook or PeerInfo struct fields change.
-    ///
-    /// Stores the current peer book to the given storage object.
-    ///
-    /// This function serializes the peer book into a byte vector for storage.
-    ///
-    #[inline]
-    pub(crate) fn save_peer_book_to_storage(&self) -> Result<(), NetworkError> {
```

Comment:
I think it's safe to say we won't want to use this function in this shape (since we don't want to keep the `PeerBook` in the storage belonging to `Sync`), so we might as well remove it for now :+1: